test:TestWatchCompactRevision

```
--- FAIL: TestWatchCompactRevision (0.17s)
	watch_test.go:428: wresp.Err() expected etcdserver: mvcc: required revision has been compacted, but got <nil>
```

--- a/clientv3/integration/watch_test.go
+++ b/clientv3/integration/watch_test.go
@@ -400,7 +400,7 @@ func TestWatchResumeCompacted(t *testing.T) {
 func TestWatchCompactRevision(t *testing.T) {
 	defer testutil.AfterTest(t)
 
-	clus := integration.NewClusterV3(t, &integration.ClusterConfig{Size: 3})
+	clus := integration.NewClusterV3(t, &integration.ClusterConfig{Size: 1})
 	defer clus.Terminate(t)
 
 	// set some keys

